### PR TITLE
fix: segment

### LINKED ISSUE #175: Fix Segment
Segmento no working

### PR DESCRIPTION
**Pull request by cazala**
Closes #175

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -34,6 +34,56 @@
...
+              n.async = !0
+              n.src = 'https://cdn.segment.com/analytics.js/v1/' + t + '/analytics.min.js'
+              var a = document.getElementsByTagName('script')[0]
+              a.parentNode.insertBefore(n, a)
+              analytics._loadOptions = e
+            }
+            analytics.SNIPPET_VERSION = '4.1.0'
+          }
+      })()
+    </script>
```
Comment (nicosantangelo) [inline public/index.html]: Maybe better minified?

Comment (cazala) [reply public/index.html]: this is as provided by segment, it’s just not in one line because of prettier

Comment (cazala) [reply public/index.html]: i could prettier-ignore it

Comment (cazala) [reply public/index.html]: done ✅

**Comment by cazala** [comment]
@nicosantangelo I added a few actions to track, it's ready for review now

**Pull request merged**